api/src/test/java/org/pmiops/workbench/test/TestDialect.java

```diff
@@ -0,0 +1,16 @@
+package org.pmiops.workbench.test;
+
+import org.hibernate.dialect.MySQLDialect;
+import org.hibernate.dialect.function.SQLFunctionTemplate;
+import org.hibernate.type.StandardBasicTypes;
+
+public class TestDialect extends MySQLDialect {
+
+  public TestDialect() {
+    super();
+    // For in-memory tests, use LIKE for full text searches.
+    registerFunction("match", new SQLFunctionTemplate(StandardBasicTypes.DOUBLE,
```

Comment:
What does this have to do with a DOUBLE? Am I missing something?

Reply:
I'm not sure why it is this way, but it is this way... see:

https://pavelmakhov.com/2016/09/jpa-custom-function

Adding a comment.